deploy/secretmanage

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python2
+"""
+secretmanage -- manage production secrets!
+
+secretmanage allows you to upload new secrets. These secrets are then available
+in production (either in real production, staging or develop) by using the
+importsecret module.
+
+
+Usage:
+    secretmanage upload (--dev|--staging|--prod) <name> <secretstring>
+
+<name> is the name of the secret. This will be used as a reference when you import
+the secret.
+
+<secretstring> is the content of the secret. These can be the actual DB credentials,
+the password to a message broker, or your life's nemesis.
+"""
+# NOTE to maintainers:
+# This encrypts the given plaintext using a keyring with the name of the environment,
+# and a key with the same name as the secret.
+# We will then upload this key to a bucket called '<project_id>-secrets/', inside a folder in that bucket
+# with the same name as the environment. The file in that bucket is named the same as
+# the secret.
+#
+# This means that you need to ensure that a keyring with the same name as the environment is created.
+# You also need to make sure that the <project_id>-secrets/ bucket is available.
+#
+# The persons running this script must also have th Encrypt permission for KMS, and has Edit and Get
+# permissions for the bucket.
+import logging
+import os
+
+from docopt import docopt
+from google.cloud import kms_v1
```

Comment:
The import failed on my local machine, so we need to include `google-cloud-kms` in the pipfile.